lib/turbo_tests/cli.rb

```diff
@@ -93,21 +98,18 @@ def run
         end
       end
 
-      success = TurboTests::Runner.run(
+      exitstatus = TurboTests::Runner.run(
         formatters: formatters,
         tags: tags,
         files: @argv.empty? ? ["spec"] : @argv,
         runtime_log: runtime_log,
         verbose: verbose,
         fail_fast: fail_fast,
         count: count,
+        test_options: test_options
       )
 
-      if success
-        exit 0
-      else
-        exit 1
-      end
+      exit exitstatus
```

Comment:
We now just report the exitstatus that comes back from the Runner directly.

Reply:
Sounds good :+1: